index.d.ts

```diff
@@ -0,0 +1,50 @@
+// Type definitions for chpr-worker 3.2.0
+// Project: chpr-worker
+// Definitions by: Chauffeur Privé
+// TypeScript Version: 3.0.1
+
+/// <reference types="node" />
+
+import { Logger } from 'chpr-logger';
+
+declare module Worker {
+    export function createWorkers(handlers: Worker.Handler[], config: Worker.Config, options?: Worker.Options): Worker.Process
+}
+
+declare namespace Worker {
```

Comment:
I wonder whether we should declare these definitions in `stakhanov` and import them here.

Reply:
Totally agree with you, that's the main idea but we didn't want to lose too much time on this for the purpose.
As we are talking about it now, what should we do?
@joana-costa @MohamedJeffal

Reply:
As seen with you both I'll do the work in stakhanov (I just hope it won't take too much time, otherwise, I'll merge the current PR to continue my current task and I will do the migration later). Thanks for the review!